service/endpoint/internal/src/main/java/org/eclipse/kapua/service/endpoint/internal/EndpointInfoServiceImpl.java

```diff
@@ -152,7 +160,12 @@ public void delete(KapuaId scopeId, KapuaId endpointInfoId) throws KapuaExceptio
 
         //
         // Check Access
-        AUTHORIZATION_SERVICE.checkPermission(PERMISSION_FACTORY.newPermission(EndpointInfoDomains.ENDPOINT_INFO_DOMAIN, Actions.delete, null));
+        KapuaId scopeIdPermission = entityManagerSession.doAction(em -> EndpointInfoDAO.find(em, scopeId, endpointInfoId))
```

Comment:
If `entityManagerSession.doAction(em -> EndpointInfoDAO.find(em, scopeId, endpointInfoId))` returns null, we have a `NullPointerException` here. Let's test this first.